### ISSUE TITLE
Need to establish guidelines for tricky arg1 dependencies

### ISSUE THREAD

**Issue by heacu** [tibetan-nlp/annotation-docs#20]
Consider the following example (see [original](http://tibetanverbs.soas.ac.uk/~badw/#/mila/020b?focus=T6888)):

![020b](https://user-images.githubusercontent.com/1461961/34588391-9d72175e-f1a4-11e7-8814-1b8cfd5e19ce.png)
> He was conscious of his own great strength and remained there without fear.

Is _kho_ really the `arg1` of _yod pa_? I suspect not, and the clue is the case marking: we have _kho_ rather than _khor_ / _kho la_, which suggests that _kho_ is probably the subject of _bzhag_, not _yod_.

**Comment by samyorode** [comment]
Of course, one could argue that _yod pa_ has a missing subject which probably would be marked with _la_ or _-r_, but I am not sure if that would always be the case. Let's see what further annotation will bring to light. So _kho_ is subject of _bzhag_ here!
<img width="444" alt="bildschirmfoto 2018-01-09 um 12 51 39" src="https://user-images.githubusercontent.com/31879546/34719676-f0da1528-f53b-11e7-83a2-7e50eb3d8dc9.png">

**Comment by heacu** [comment]
I agree that we can't take for granted that the subject of _yod pa_ must be marked with _la_ or _-r_. Moreover, if it must be, then that's something we should like to establish by corpus statistics and not just by assumption. So I'll accept the change but keep this issue open, in case other similar examples pop up that might help us.
